tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -727,10 +727,25 @@ test.describe( 'Nested Tabs tests', () => {
 		await page.waitForSelector( '.elementor-widget-n-tabs' );
 		// Test on desktop.
 		await expect( page.locator( '.e-normal.e-active' ) ).toHaveClass( 'e-n-tab-title e-normal elementor-animation-grow e-active' );
+
+		// Test the hover animation.
+		await page.locator( '.e-normal:not( .e-active )' ).last().hover();
+		const tabHover = await page.locator( '.e-normal:not( .e-active )' ).last().evaluate( ( element ) => {
+			const animationValue = window.getComputedStyle( element ).getPropertyValue( 'transform' );
+
+			return animationValue.includes( 'matrix(' ) ? true : false;
+		} );
+		await expect( tabHover ).toBe( true );
+
+		// Test the active tabs with hover.
+		await page.locator( '.e-normal.e-active' ).hover();
+		await expect( page.locator( '.e-normal.e-active' ) ).toHaveCSS( 'transform', 'none' );
+
+		// Test on mobile.
 		await page.setViewportSize( viewportSize.mobile );
 		await expect( page.locator( '.e-collapse.e-active' ) ).toHaveClass( 'e-n-tab-title e-collapse elementor-animation-grow e-active' );
-		await page.setViewportSize( viewportSize.desktop );
```

Comment:
What is the reason for this change?
Did you move this line down just for the file order?

Reply:
Changed.